pkg/ui/assets_embed.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build builtinassets
+// +build builtinassets
+
+package ui
+
+import (
+	"net/http"
+
+	"github.com/prometheus/common/assets"
+)
+
+var Assets = http.FS(assets.New(EmbedFS))
```

Comment:
My IDE isn't able to let me see what this stuff is. Are you able to navigate around here? I'm not able to find `EmbedFS` anywhere. I imagine this is some autogenerated byte payload in a go file somewhere...

In general, I'm having a hard time remembering this build flow. Could you ensure https://github.com/GoogleCloudPlatform/prometheus-engine/blob/freinartz/updateprometheus/pkg/ui/README.md#L26 is up-to-date with a lite description of the build flow?

Reply:
I was confused as well. Embedding is now a new-ish feature of Go itself. So it happens when compiling not by pre-generating a file containing all embedded contents like before.
The generated `embed.go` file merely contains a list of files to include.

README has indeed been stale for a bit. Updated it and removed a lot of stuff that's not really relevant for our development flow anymore.

Reply:
Thanks